classes/user/UserDAO.inc.php

```diff
@@ -513,9 +519,11 @@ function userExistsByUsername($username, $userId = null, $allowDisabled = true)
 	 * @return boolean
 	 */
 	function userExistsByEmail($email, $userId = null, $allowDisabled = true) {
+		$emailLower = strtolower($email);
```

Comment:
Mixing PHP-side string operations with database-side string operations may cause trouble with locales and collations. I'd recommend keeping everything in one place -- e.g. `LOWER(?)` in the database query.